index/index.go

```diff
@@ -0,0 +1,13 @@
+package index
+
+import (
+	"github.com/kuzzleio/sdk-go/types"
+)
+
+type Index struct {
+	k types.IKuzzle
```

Comment:
I'm not fond of 1-letter variable names, especially for interface properties, as they needlessly make the code harder to read.